### PR TITLE
Create a skeleton for dappkit and contractkit documentation

### LINKED ISSUE #487: Create an SDK Documentation Skeleton
### Expected Behavior
Add a new section on SDK to docs.celo.org
- DappKit Intro
- Examples Gallery for dAppKit

### Current Behavior

No docs yet

### PR DESCRIPTION
**Pull request by nityas**
### Description

This pull request adds the skeleton of documentation for dappkit and contractkit, to be filled in by various contributors

### Tested

I've copied the structure used by the `celo-codebase` section, seeking guidance on how to see a preview with these changes.

### Other changes

n/a

### Related issues

- Fixes #487 

### Backwards compatibility

Backwards compatible.

### CODE REVIEW

File: packages/dappkit/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  })
+
+  const dappkitResponse = await waitForAccountAuth(requestId)
+
+  this.setState({ address: dappkitResponse.address, phoneNumber: dappkitResponse.phoneNumber })
+}
+```
+
+Once you have the account address, you can make calls against your own smart contract, or use [ContractKit](../contractkit/README.md) to fetch a users balance
+
```
Comment (timmoreton) [inline packages/dappkit/README.md:51]: Typo: user’s

Comment (timmoreton) [reply packages/dappkit/README.md:51]: And add a colon

File: packages/dappkit/README.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  const [cUSDBalanceBig, cUSDDecimals] = await Promise.all([stableToken.balanceOf(address), stableToken.decimals()])
+  const cUSDBalance = this.convertToContractDecimals(cUSDBalanceBig, cUSDDecimals)
+
+  this.setState({ cUSDBalance, isLoadingBalance: false })
+```
+
+## Get Contact List Addresses
+
+For many real-world applications, your user will want to interact with their friends and family on your DApp. Celo has a built-in [Identity Protocol](../../celo-codebase/protocol/identity/README.md) that maps phone numbers to account addresses. You can use DAppkit to fetch that mapping for a users contact list.
```
Comment (timmoreton) [inline packages/dappkit/README.md]: Typo: user’s

File: packages/dappkit/README.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+Object.entries(this.state.phoneNumbersByAddress).map(([address, entry]) => {
+  const contact = this.state.rawContacts[entry.id]
+  return (
+    <Button key={address} title={contact.name} onPress={() => {}} />
+  )
+})
+```
+
+## Signing Transactions
+
```
Comment (timmoreton) [inline packages/dappkit/README.md:98]: read-only -> accessing account information

Comment (timmoreton) [reply packages/dappkit/README.md:98]: Capitalize ContractKit

Comment (timmoreton) [reply packages/dappkit/README.md:98]: and DAppLit

File: packages/docs/developer-resources/dappkit/README.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+DAppKit is a lightweight set of functions that allow mobile DApps to work with the Celo Wallet to sign transactions and access the user's account. This allows for a better user experience: DApps can focus on a great native experience without having to worry about key management. It also provides a simpler development experience, as no state or connection management is necessary.
 
 DAppKit supports the following functionality:
 
 - Request permission to access account information and phone number from the Celo Wallet
 - Request permission to sign transaction(s) from the Celo Waller
 - Look up phone numbers using the [Identity Protocol](../../celo-codebase/protocol/identity/README.md) to find contacts using Celo.
 
+DAppKit is currently built with the excellent [Expo framework](https://expo.io) in mind. In the near future, we will make it more generic to all of React Native and possibly native stacks, but for now you get to take advantage of some awesome features like an incredibly easy setup, hot-reloading, and more.
```
Comment (timmoreton) [inline packages/docs/developer-resources/dappkit/README.md:11]: Sounds like you’re selling their product!

File: packages/docs/developer-resources/dappkit/setup.md [hunk 5]
```diff
@@ -1,3 +1,100 @@
...
+yarn global add expo-cli
+
+expo init $YOUR_APP_NAME
+```
+
+Read more about general Expo setup at [https://expo.io/learn](https://expo.io/learn)
+
+## Typescript Support
+
+We are big fans of Typescript, so if you used the tabs template, you can support typescript by just following [these guide lines](https://docs.expo.io/versions/latest/guides/typescript/)
```
Comment (timmoreton) [inline packages/docs/developer-resources/dappkit/setup.md]: guidelines

File: packages/docs/developer-resources/dappkit/setup.md [hunk 6]
```diff
@@ -1,3 +1,100 @@
...
+
+To add DAppKit, simply add `@celo/dappkit@0.0.5` to your package.json or
+
+```bash
+npm install @celo/dappkit@0.0.6
+// or
+yarn add @celo/dappkit@0.0.6
+```
+
+DAppkit's dependencies require a bit of adjustment to a vanilla Expo. The first are a lot of the Node.js modules that are expected. You can get those mostly by using the following modules
```
Comment (timmoreton) [inline packages/docs/developer-resources/dappkit/setup.md]: DAppKit

File: packages/docs/developer-resources/dappkit/usage.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
+# Usage
+
```
Comment (timmoreton) [inline packages/docs/developer-resources/dappkit/usage.md:2]: DAppKit

File: packages/docs/developer-resources/dappkit/usage.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+This page walks you through the main functionalities of dappkit. You can also find the result of this walkthrough on the [expo base template](https://github.com/celo-org/dappkit-base) on branch [`dappkit-usage`](https://github.com/celo-org/dappkit-base/tree/dappkit-usage).
+
+DAppKit uses deeplinks to communicate between your DApp and the Celo wallet. All "requests" that your DApp makes to the wallet needs to contain the follwing meta payload:
```
Comment (timmoreton) [inline packages/docs/developer-resources/dappkit/usage.md]: Wallet

**Review by nambrot — APPROVED** [review]

**Pull request merged**